libs/sdk-ui-kit/src/Dialog/ConfirmDialogBase.tsx

```diff
@@ -67,6 +67,7 @@ export const ConfirmDialogBase = React.memo<IConfirmDialogBaseProps>(function Co
             accessibilityConfig={accessibilityConfig}
             initialFocus={initialFocus}
             returnFocusTo={returnFocusTo}
+            returnFocusAfterClose={true}
```

Comment:
(this is not a request for change)

Why not just
```suggestion
            returnFocusAfterClose
```
style preference?